.circleci/config.yml

```diff
@@ -38,4 +38,93 @@ jobs:
           destination: reports
       - store_artifacts:
           path: build/reports/jacoco/jacocoFullReport/
+  push_translations:
+    docker:
+      - image: circleci/python:3.6
+    working_directory: ~/ultrasonic
+    steps:
+      - checkout
+      - run:
+          name: install transifex client
+          command: |
+            python -m venv ~/venv
+            . ~/venv/bin/activate
+            pip install transifex-client
+      - run:
+          name: configure transifex client
+          command: echo $'[https://www.transifex.com]\nhostname = https://www.transifex.com\nusername = api\npassword = '"${TRANSIFEX_PASSWORD}"$'\n' > ~/.transifexrc
+      - run:
+          name: push changes in translation files
+          command: |
+            . ~/venv/bin/activate
+            tx push -st
+  generate_signed_apk:
+    docker:
+      - image: circleci/android:api-27-alpha
+    working_directory: ~/ultrasonic
+    envoronment:
+      JVM_OPTS: -Xmx3200m
+    steps:
+      - checkout
+      - restore_cache:
+          key: gradle-cache-{{ checksum "dependencies.gradle" }}
+      - run:
+          name: decrypt ultrasonic-keystore
+          command: openssl aes-256-cbc -K ${ULTRASONIC_KEYSTORE_KEY} -iv ${ULTRASONIC_KEYSTORE_IV} -in ultrasonic-keystore.enc -out ultrasonic-keystore -d
+      - run:
+          name: build release apk
+          command: ./gradlew build assembleRelease
+      - run:
+          name: sign release apk
+          command: |
+            mkdir -p /tmp/ultrasonic-release
+            cd ultrasonic/build/outputs/apk/release/
+            for apk in *.apk; do
+              jarsigner -verbose -sigalg SHA1withRSA -digestalg SHA1 -keystore ~/ultrasonic/ultrasonic-keystore -storepass ${ULTRASONIC_KEYSTORE_STOREPASS} -keypass ${ULTRASONIC_KEYSTORE_KEYPASS} ${apk} ultrasonic
+              jarsigner -verify ${apk}
+              ${ANDROID_HOME}/build-tools/27.0.0/zipalign -v 4 ${apk} /tmp/ultrasonic-release/${apk/-release/}
+            done
+      - persist_to_workspace:
+          root: /tmp/ultrasonic-release
+          paths:
+            - ultrasonic-*.apk
+  publish_signed_apk:
```

Comment:
Can you rename to `publish_github_release`? Because initially I though that it publishes to play store.

Reply:
Done! Better `publish_github_signed_apk` that is more descriptive ;-)